packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -503,6 +506,7 @@ class FlutterSkippingGoldenFileComparator extends FlutterGoldenFileComparator {
   /// skipped.
   static bool isAvailableForEnvironment(Platform platform) {
     final String luci = platform.environment['SWARMING_TASK_ID'] ?? '';
-    return luci.isNotEmpty;
+    final String cirrus = platform.environment['CIRRUS_CI'] ?? '';
+    return luci.isNotEmpty || cirrus.isNotEmpty;
```

Comment:
`platform.environment.containsKey('CIRRUS_CI')` (et al) is probably more idiomatic.